keras_cv/models/serialization_test.py

```diff
@@ -0,0 +1,234 @@
+            DarkNet53, 
+            {
+                "include_rescaling": True,
+                "include_top": True,
+                "classes" : 4
+            }
+        ),
+
+        (
+            "DenseNet121", 
+            DenseNet121, 
+            {
+                "include_rescaling": True,
+                "include_top": True,
+                "classes" : 4
+            }
+        ),
+
+        (
+            "DenseNet169", 
+            DenseNet169, 
+            {
+                "include_rescaling": True,
+                "include_top": True,
+                "classes" : 4
+            }
+        ),
+
+        (
+            "DenseNet201", 
+            DenseNet201, 
+            {
+                "include_rescaling": True,
+                "include_top": True,
+                "classes" : 4
+            }
+        ),
+
+        (
+            "MLPMixerB16", 
+            MLPMixerB16, 
+            {
+
+            }
+        ),
+
+        (
+            "MLPMixerB32", 
+            MLPMixerB32, 
+            {
+
+            }
+        ),
+
+        (
+            "MLPMixerL16", 
+            MLPMixerL16, 
+            {
+
+            }
+        ),
+
+        (
+            "ResNet50", 
+            ResNet50, 
+            {
+                "include_rescaling": True,
+                "include_top": True,
+                "classes" : 4
+            }
+        ),
+
+        (
+            "ResNet101", 
+            ResNet101, 
+            {
+                "include_rescaling": True,
+                "include_top": True,
+                "classes" : 4               
+            }
+        ),
+
+        (
+            "ResNet152", 
+            ResNet152, 
+            {
+                "include_rescaling": True,
+                "include_top": True,
+                "classes" : 4
+            }
+        ),
+
+        (
+            "ResNet50V2", 
+            ResNet50V2, 
+            {
+                "include_rescaling": True,
+                "include_top": True,
+                "classes" : 4
+            }
+        ),
+
+        (
+            "ResNet101V2", 
+            ResNet101V2, 
+            {
+                "include_rescaling": True,
+                "include_top": True,
+                "classes" : 4
+            }
+        ),
+
+        (
+            "ResNet152V2", 
+            ResNet152V2, 
+            {
+                "include_rescaling": True,
+                "include_top": True,
+                "classes" : 4
+            }
+        ),
+        
+        (
+            "VGG19", 
+            VGG19, 
+            {
+                "include_rescaling": True,
+                "include_top": True,
+                "classes" : 4
+            }
+        )
+    )
+    def test_model_serialization(self, model_cls, init_args):
+        model = model_cls(**init_args)
+        config = model.get_config()
+        self.assertAllInitParametersAreInConfig(model_cls, config)
+
+        test_model = model
```

Comment:
any reason to re-assign model to a new parameter here?  This is basically a no-op.

Reply:
removed reassignment and used the same model.